components/molecules/dropdown/components/item.js

```diff
@@ -27,18 +27,62 @@ const renderItem = ({ id, name, to }, closeDropdown, onClick) => {
   )
 }
 
-export const DropdownItem = ({ closeDropdown, item, onClick }) => {
-  return (
-    <StyledDropdownItem divider={item.divider}>
-      {item.divider ? <StyledDivider /> : renderItem(item, closeDropdown, onClick)}
-    </StyledDropdownItem>
-  )
+export const DropdownItem = ({ closeDropdown, elementType, item, onClick }) => {
+  switch (elementType) {
+    case elementTypes.List:
+      return (
+        <StyledDropdownItem divider={item.divider}>
+          {item.divider ? <StyledDivider /> : renderItem(item, closeDropdown, onClick)}
+        </StyledDropdownItem>
+      )
+    case elementTypes.Colour:
+      return (
+        <StyledColourItem
+          colour={item.colour}
+          onClick={() => {
+            onClick(item)
+          }}
+        />
+      )
+    default:
+      return 'invalid elementType'
+  }
 }
+const StyledColourItem = styled.div`
+  width: 1.55rem;
+  height: 1.55rem;
+  ${({ colour }) =>
+    css`
+      background-color: ${colour};
+    `}
+  ${({ colour }) => {
+    const size = '0.5rem'
+    return (
+      colour === 'transparent' &&
+      css`
+        background-image: linear-gradient(45deg, #808080 25%, transparent 25%),
+          linear-gradient(-45deg, #808080 25%, transparent 25%),
+          linear-gradient(45deg, transparent 75%, #808080 75%),
+          linear-gradient(-45deg, transparent 75%, #808080 75%);
+        background-size: ${size} ${size};
+        background-position: 0 0, 0 ${size}, ${size} -${size}, -${size} 0px;
+      `
+    )
+  }}
+
 
+  &&:hover {
+    border: 0.15rem solid white;
```

Comment:
@mhsnprvr2 Colours we tend to always use the CONTEXT colours - theme.COLOUR.white